use_cases.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Use Cases"
+draft: false
+---
+
+# Use Cases
+
+## Internet of Things
+
+YottaDB fits the needs of real-time IoT systems that need to be “always on” because of its proven availability, reliability and security. Because it is highly scalable and multi-model, it can also handle the varying requirements of application data with ease.
+
+Plus, using a single database engine for the edge and the cloud simplifies application design and operational management, as well as allowing data to move seamlessly from one instance to another.
+
+## Health Records
+
+[M](https://en.wikipedia.org/wiki/MUMPS) is a de facto standard for implementing electronic health record applications, and the largest electronic health record systems in the world are written in M, dominating the segment as effectively as SQL dominates accounting and business applications.
+
+Under the GT.M brand, the YottaDB code base has its roots in healthcare as an implementation of M, with its first deployment at the Elvis Presley Trauma Center in Memphis in 1986. YottaDB includes an M language subsystem that is mostly compliant with, and extends the ISO/IEC 11756:1999 and ISO/IEC 15851:1999 standards. The YottaDB/GT.M code base is one of the standard platforms for implementing VistA, ranging from small primary care practices and community clinics such as Oroville Hospital to Hakeem, an electronic health record system for an entire country.
```

Comment:
Under the GT.M brand, the YottaDB → The YottaDB

The YottaDB/GT.M code base → The code base